ecommerce/tests/test_models.py

```diff
@@ -0,0 +1,49 @@
+from ecommerce.tests.factories import CategoryFactory, ProductFactory
+from django.test import TestCase
```

Comment:
Thứ tự thư viện của local sau e nha
```suggestion
from django.test import TestCase
from ecommerce.tests.factories import CategoryFactory, ProductFactory
```